Cargo.toml

```diff
@@ -38,17 +38,23 @@ std = ["alloc"]
 alloc = ["race"]
 
 # Enables `once_cell::race` module.
-race = []
+race = ["portable-atomic?/require-cas"]
```

Comment:
Hmm, due to Cargo bug (https://github.com/rust-lang/cargo/issues/10801), this makes portable-atomic to appear in Cargo.lock (even if the portable-atomic feature is not enabled. even if portable-atomic will never be compiled).

With the diagnostic improvements in portable-atomic 1.8 (https://github.com/taiki-e/portable-atomic/pull/181), the need for require-cas feature has decreased, so I think it is probably fine to remove `"portable-atomic?/require-cas"`...

Reply:
I am also fine with keeping it! Cargo bug is a bug, the code here seems correct, and could perhaps be helpful for older versions of portable-atomic? 

I guess you are the best person to make a judgement call here though :)